alembic/versions/938e21c8c260_add_inappropriate_content_column.py

```diff
@@ -0,0 +1,63 @@
+"""Add inappropriate content column
+
+Revision ID: 938e21c8c260
+Revises: 826e6c309c31
+Create Date: 2017-08-31 16:17:50.420672
+
+"""
+
+
+# revision identifiers, used by Alembic.
+revision = '938e21c8c260'
+down_revision = '5c14f5a350dd'
```

Comment:
I manually edited the `down_revision` here to point to #96's revision, so it'd end up as a linear path.